python/test/test_api.py

```diff
@@ -343,7 +346,7 @@ def test_ready(self):
         self.assertTrue(server.ready())
 
     @pytest.mark.xfail(
-        tritonserver.__version__ <= "2.42.0",
+        tritonserver.__version__ <= "2.43.0",
```

Comment:
I see we sequentially changing version. <br>
Is this something that will have place in for version `2.44.0`?

Reply:
Not yet scheduled / prioritized.